package.json

```diff
@@ -8,7 +8,9 @@
     "eslint": "node_modules/eslint/bin/eslint.js \"config.js\" \"features/**/*.js\" \"index.*.js\"",
     "start:android": "npm run clean; react-native run-android",
     "start:ios": "npm run clean; react-native run-ios",
-    "start:web": "webpack-dev-server --inline --hot --colors --port 5000 --https --history-api-fallback --open"
+    "start:web": "webpack-dev-server --inline --hot --colors --port 5000 --https --history-api-fallback --open",
+    "lint": "eslint .",
+    "validate": "npm ls"
```

Comment:
This is something that "npm install/update" adds if you have precommit-hook inside node_modules. Even if you don't reference it inside package.json. Can be safely removed.